### DISCUSSION TITLE
Remove specified items from menu bar menu

### DISCUSSION THREAD

**Discussion by systwi-again** [rxhanson/Rectangle#543] [Ideas]
Hello, I use Rectangle solely for moving windows back and forth between my monitors. While the resizing and positioning features do look nice, I don't really use them and prefer to keep them disabled. If you take a quick look in Rectangle's menu bar menu, you can see the list is quite long, with "Next Display" and "Previous Display" near the bottom.

It would be nice if I could hide the window resizing buttons and only have:
- Next Display
- Previous Display
- Ignore "AppName"
- Preferences...
- About
- Check for Updates...
- Quit Rectangle

available in the list.

If support for this could be added to Rectangle I would greatly appreciate it!

**Comment by rxhanson** [comment]
Being able to customize what’s in the menu bar is one of the items that I was planning for the next release. At first cut, it would be a terminal command setting. 

I’m currently working through an injury, so it’s going to take a while for me to get to it. If anyone else wants to make a PR for it in the meantime, I’d gladly review it.
